src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Values/CultureValue.cs

```diff
@@ -0,0 +1,43 @@
+using System.Globalization;
+using System.Text.Encodings.Web;
+using Fluid;
+using Fluid.Values;
+using OrchardCore.Localization;
+
+namespace OrchardCore.DisplayManagement.Liquid.Values;
+
+public class CultureValue : FluidValue
+{
+    private readonly CultureInfo _culture;
+
+    public CultureValue(CultureInfo culture) => _culture = culture;
+
+    public override FluidValues Type => FluidValues.Object;
+
+    public override bool Equals(FluidValue other)
+    {
+        throw new NotImplementedException();
+    }
+
+    public override bool ToBooleanValue() => false;
+
+    public override decimal ToNumberValue() => 0;
+
+    public override object ToObjectValue() => ToStringValue();
+
+    public override string ToStringValue() => _culture.Name;
+
+    public override void WriteTo(TextWriter writer, TextEncoder encoder, CultureInfo cultureInfo)
```

Comment:
`WriteToAsync`?

Reply:
I wrote another PR while you writing comments :)

https://github.com/sebastienros/fluid/pull/709